R/int_pctl.R

```diff
@@ -147,13 +151,30 @@ boostrap_metrics_by_config <- function(config, seed, x, metrics, times, allow_pa
   set.seed(seed)
   rs <- rsample::bootstraps(preds, times = times)
 
-  `%op%` <- get_int_p_operator(allow_par)
+  do_op <- get_int_p_operator(allow_par)
+  `%op%` <- do_op[[1]]
+  is_future <- do_op[[2]]
+
+  # Rather than generating them programmatically, write each `foreach()`
+  # call out since `foreach()` `substitute()`s its dots. Note that
+  # doFuture will error when passed `.packages`.
+  if (is_future) {
+    for_each <-
+      foreach::foreach(
+        i = 1:nrow(rs),
```

Comment:
not that it matters a lot

```suggestion
        i = seq_len(nrow(rs)),
```